eduNEXT/edunext-platform
# FEC-24: Allow Instructors to manage certificates

## Description

1. Allow for Instructors (Course Admins) to manage certificates via the instructor tab.
2. Remove css that hides the signatures.

### Previous commit
[`c6511ac`](https://github.com/eduNEXT/edunext-platform/pull/421/commits/c6511ac1eb4565cbf87af4e895122a1a377bbb7d)

## Testing instructions

1. Create a course with a certificate
2. Create a new user without special permissions (superuser, staff).
2. Add the user to the course team as Admin (i.e. Instructor) via the instructor tab.
3. The certificate tab should be visible and all the certificate operations should work (certificate generation, whitelist, self generation, etc)

## Aditional Info

FEC-24 in Juniper also added a function that updated the certificate context with some defaults. This was done to avoid a mako exception when some variables were not set

```python
def _update_minimal_context(context):
    """
    This is an eduNEXT function to make the regular certificate work out of the box
    """
    context['company_privacy_url'] = ''
    context['logo_src'] = ''
    context['company_tos_url'] = ''
    context['company_about_url'] = ''
    context['logo_url'] = ''
```
Since https://github.com/edx/edx-platform/pull/25872 the default template doesn't cause an error when rendering even when not setting any of the variables, so I decided to not include the code.
I also didn't include the image [`ico-audit.png`](https://github.com/eduNEXT/edunext-platform/blob/c6511ac1eb4565cbf87af4e895122a1a377bbb7d/lms/static/certificates/images/ico-audit.png). As far as I'm aware, audit modes don't offer certificates and the only way to have this image appear would be by appending `?mode=audit` to the certificate preview. The image is rendered [here](https://github.com/eduNEXT/edunext-platform/blob/open-release/lilac.master/lms/templates/certificates/_accomplishment-rendering.html#L19).
